deployment/data-streams/changeset/reward-manager/state.go

```diff
@@ -0,0 +1,52 @@
+package reward_manager
+
+import (
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/common"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+	"github.com/smartcontractkit/chainlink/deployment/data-streams/changeset/types"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/llo-feeds/generated/reward_manager_v0_5_0"
+)
+
+func LoadRewardManagerState(
```

Comment:
This function doesn't seem to be called. Instead, we're calling https://github.com/smartcontractkit/chainlink/blob/193665e0e68bc835f7d37cd9e1e18a4476179a8c/deployment/data-streams/changeset/reward-manager/general/reward_manager.go#L13-L13. Is this intentional or do we need to remove one of these?

Reply:
left over - removed thanks for finding